src/index.js

```diff
@@ -581,6 +581,13 @@ export function app(props) {
     dispatch(event.currentTarget.events[event.type], event)
   }
 
+  if (ENVIRONMENT === 'development') {
```

Comment:
Does Uglify have dead code elimination on to actually remove this from the bundle? How about the `makeDebugger` function?

Reply:
Looks like it, but I'll attach the build files so we can all double check

Reply:
@okwolf @jorgebucaran here's the build outputs: https://gist.github.com/mrozbarry/b42ccf9704e7eee5ffc281b72f384440

We'll want to double check that makeDebug is not present in hyperapp.js or hyperapp.min.js, but is in hyperapp.dev.js